changelog.md

```diff
@@ -1,9 +1,52 @@
 # Changelog
 
-## 1.8.0~rc1
+## 1.9.0~rc1
 
 *
 
+## 1.8.0
+
+### Web applications
+
+* Provide end-of-life messaging and disable source interface after Xenial End-of-life (#5789)
+* Adds safe deletion functionality to the Journalist Interface (#5770, #5827)
+* source\_app.utils.normalizer\_timestamps will no longer create an empty file (#5724)
+
+### Operations
+
+* Support Ubuntu 20.04 Focal for SecureDrop server installs (#4728)
+* Update and add annotations to SSH configuration for servers (#5666)
+* (Focal) Disable v2 onion addresses on restore (#5677)
+* (Focal) Replace cron-apt with unattended-upgrades (#5684)
+* BUGFIX: Only configure v2 ssh service when v2 services are enabled (#5718)
```

Comment:
Just a note that this was in fact released with 1.7.0 (03c66565f43a9ce8574308fd006ec30e8cb60830). (We omitted it in the public release notes for that reason.)

Reply:
removed